### ISSUE TITLE
Akregator bug: Plugin library 'kde_grantlee_plugin' not found.

### ISSUE THREAD

**Issue by Poetastrophe** [NixOS/nixpkgs#96255]
**Describe the bug**
The news aggregator will not show the content of any news-feed, and will instead report the error in the title.

**To Reproduce**
`$ nix-shell -p akregator`
`$ akregator`
then click on any news-feed.

**Expected behavior**
To see the content of the news-feed

**Screenshots**
![kdeplugin](https://user-images.githubusercontent.com/17691932/91154862-55750e00-e6c2-11ea-8113-3a0e19a8a295.png)

**Notify maintainers**

@vandenoever 
 
**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.42, NixOS, 20.03.1998.93b68c7cc5a (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.5`
 - channels(chris): `"home-manager"`
 - channels(root): `"nixos-20.03.1998.93b68c7cc5a"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by r-k-b** [comment]
Might be fixed a newer version? <https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=972801>

**Comment by liketechnik** [comment]
I don't think so. I've hit this together with the report in https://github.com/NixOS/nixpkgs/issues/41426#issuecomment-1001985848, after switching from X to wayland (without updating/changing any configuration).
